### PR TITLE
Adding an explanation of the branching strategy

### PR DESCRIPTION
**Pull request by LindsayYoung**
Documentation update

## What does this change?
Working on documenting the release flow

## Checklist:

+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests)

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: docs/Branching_strategy.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+## Branching strategy:
+
+This strategy is aimed at reducing the risk of production deploys. Smaller, more frequent deploys reduce the risk of each deploy. Since these smaller deploys are less likely to cause problems, that provides the benefit of less downtime and higher availability of your systems. It also has the benefit of better schedules for maintainers since there is rarely a need to wait until the site is not in use if you can do seamless deploys.
+
+So, the overall goal is to make sure that we have small, frequent deploys while ensuring proper quality checks. Before code is deployed it goes though 3 human evaluations and at least 3 rounds of automated testing.
+
+We currently have automated tests for:
+        - accessibility
```
Comment (alexsoble) [inline docs/Branching_strategy.md]: Noticing that this isn't rendering as a list in Markdown.

File: docs/Branching_strategy.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+## Branching strategy:
+
+This strategy is aimed at reducing the risk of production deploys. Smaller, more frequent deploys reduce the risk of each deploy. Since these smaller deploys are less likely to cause problems, that provides the benefit of less downtime and higher availability of your systems. It also has the benefit of better schedules for maintainers since there is rarely a need to wait until the site is not in use if you can do seamless deploys.
+
+So, the overall goal is to make sure that we have small, frequent deploys while ensuring proper quality checks. Before code is deployed it goes though 3 human evaluations and at least 3 rounds of automated testing.
```
Comment (alexsoble) [inline docs/Branching_strategy.md]: This paragraph uses "we" but the paragraph above uses "you", we might want to keep that consistent for clarity.

**Comment by LindsayYoung** [comment]
fixed the markdown and got rid of "we" and changed it to team.

**Comment by alexsoble** [comment]
Looks great!! 👍👍

**Pull request merged**